src/pages/sponsor/[uid].astro

```diff
@@ -21,22 +22,157 @@ const descriptionHTML = prismicH.asHTML(sponsor.data.description);
 ---
 
 <Layout
-  title={`${sponsor.data.name} - Guest Speaker - Holiday Snowtacular`}
+  title={`${sponsor.data.name} - Sponsor - Holiday Snowtacular`}
   description={`${sponsor.data.name} is sponsoring the 2022 Holiday Snowtacular in an effort to raise $20,000 for Doctors Without Borders!`}
 >
-  <main>
-    <h1>{sponsor.data.name}</h1>
-    <img src={sponsor.data.logo.url} alt="" />
-    <Fragment set:html="descriptionHTML" />
+  <main class="holiday-supporter">
+    <div class="holiday-supporter-heading">
+      <span class="holiday-supporter-eyebrow">Sponsor</span>
+      <h1 class="holiday-supporter-name">{sponsor.data.name}</h1>
+    </div>
+    <div class="holiday-supporter-logo-container">
+      <img src={sponsor.data.logo.url} alt={`${sponsor.data.name} logo`} />
+    </div>
+    <div class="holiday-supporter-description">
+      <Fragment set:html="descriptionHTML" />
+    </div>
 
     {
-      prismicH.isFilled.link(sponsor.data.link) && (
-        <p>
-          <a href={sponsor.data.link.url}>Visit Site</a>
-        </p>
+      prismicH.isFilled.link(sponsor.data.ctalink) && (
+        <div class="holiday-supporter-cta">
+          <a class="button" data-variant="red" href={sponsor.data.ctalink.url}>
```

Comment:
Yeah good call there. Should this also have a `target="_blank"` ?

Reply:
I usually go for it just to keep the page itself open should the user desire any further interaction

Reply:
From an accessibility perspective, `target="_blank"` isn't recommended, since it can disorient screenreader users and just overall decreases user agency. I'm not particularly worried about any of the security concerns folks usually cite for `_blank` in this case, since we trust our sponsors.

Reply:
For more context, see [Adrian Roselli's post on link targets](https://adrianroselli.com/2020/02/link-targets-and-3-2-5.html).